community-membership.md

```diff
@@ -113,26 +113,26 @@ only be doing issues triage and have no approval rights.
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
-- Reviewer of the codebase for at least 3 months
+- Reviewer of the codebase for at least 1 month
 - Reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Reviewed or merged at least 20 PRs to the codebase
```

Comment:
```suggestion
- Reviewed at least 20 PRs to the codebase
```
approvers cannot merge

Reply:
I think the phrasing was meant to mean "reviewed someone else's pull, or had their own pull merged by someone else..."

Reply:
PTAL, I just updated the wording.